content/docs/latest/spire/installing/getting-started-k8s.md

```diff
@@ -27,7 +27,9 @@ The steps in this guide have been tested on these Kubernetes versions: 1.13.1, 1
 {{< info >}}
 If you are using Minikube to run this tutorial you should specify some special flags as described [here](#considerations-when-using-minikube).
 {{< /info >}}
-
+{{< info >}}
+If you are using Kubeadm to run this tutorial, you should create the resources specified [here](#considerations-when-using-kubeadm).
```

Comment:
I'm not sure if this is 100% correct, because we are creating a storage class and a storage provisioner to bind a claim to an automatically created persistent volume. Maybe we can write something like:
```suggestion
If you are using Kubeadm to run this tutorial, you should manually create the resources needed to provide a persistent volume, as described [here](#considerations-when-using-kubeadm).
```
What do you think?

Reply:
👍  Yes, that sounds good. I'm glad to have it as accurate as possible.